stdlib/public/core/CollectionOfOne.swift

```diff
@@ -41,7 +43,17 @@ extension IteratorOverOne: IteratorProtocol, Sequence {
   }
 }
 
-/// A collection containing a single element of type `Element`.
+/// A collection containing a single element.
+///
+/// You can use a `CollectionOfOne` instance to efficiently represent a
```

Comment:
Is there a way to reword this so it isn't quite as repetitive of the line above?

Reply:
I'll take a crack at this 👍